source/timemory/components/timing/backends.hpp

```diff
@@ -78,6 +80,7 @@ gettimeofday(struct timeval* t, void* timezone) noexcept
     t->tv_usec = 1000 * timebuffer.millitm;
     return 0;
 }
+#endif
```

Comment:
Why is `gettimeofday` blocked out here?

Reply:
Because it uses the ``timeval`` struct, which seems to be available on windows only through ``winsock.h``. 
I could see no other way to fix it. Since ``gettimeofday`` does not seem to be used anywhere else, I thought it was okay.